src/components/CreateToken/ConfirmTokenCreationModal.tsx

```diff
@@ -43,100 +52,175 @@ const InformationRow = styled(Columns)`
 `;
 
 const Bottom = styled.div`
-  padding-top: 70px;
+  margin-top: auto;
+  padding-top: 5px;
   ${Button} {
     margin-top: 12px;
   }
 `;
 
+const CustomAttributesDivider = styled.div`
+  margin-top: 10px;
+  margin-bottom: 25px;
+  border-bottom: 1px dashed ${V.color.frontSoft};
+  width: 70%;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
 const NotApplicable = () => <i>N/A</i>;
 
 const ConfirmTokenCreationModal: React.FC = () => {
   const token = useSelector(selectModalOptions) as TokenForm;
 
-  const tokenTypeName = useMemo(() => (token.supply === 1 ? 'NFT' : 'token'), [token]);
+  const tokenCreationSchema = useTokenCreationSchema();
+
+  const tokenType = useMemo(() => (token.supply === 1 ? TokenType.NFT : TokenType.TOKEN), [token]);
```

Comment:
I'm not a huge fan of this from a syntax/separation of concerns perspective, but understand the performance aspect of this choice. We should make a note of it anyhow in Notion.

Reply:
Yep it does start to get a little concentrated but I think if you're using `useMemo` you gotta prioritize perf over maximally clean code, otherwise may as well not use `useMemo` at all.